libvirt/tests/src/storage/virsh_pool_vol.py

```
@@ -0,0 +1,405 @@
+    pool_defined = False
+    pool_started = False
+    vol_created = False
+    fs_source_dev = None
+
+    try:
+        if not os.path.isdir(pool_target):
+            logging.info("Creating pool target directory: %s", pool_target)
+            os.makedirs(pool_target)
+        pool_define_extra = ""
+        if pool_type == "fs":
+            root_src = process.run(
```

Ruff (0.15.20)

[error] Function call with `shell=True` parameter identified, security issue

(S604)